lambdas/check-coi/src/main/java/uk/gov/di/ipv/core/checkcoi/CheckCoiHandler.java

```diff
@@ -104,7 +104,11 @@ public Map<String, Object> handleRequest(ProcessRequest request, Context context
                                 .areFamilyNameAndDobCorrelatedForCoiCheck(credentials);
                         case FAMILY_NAME_ONLY, FAMILY_NAME_AND_ADDRESS -> userIdentityService
                                 .areGivenNamesAndDobCorrelated(credentials);
-                        case ADDRESS_ONLY -> userIdentityService.areVcsCorrelated(credentials);
+                        case ADDRESS_ONLY -> {
+                            LOGGER.warn(
+                                    LogHelper.buildLogMessage("Address only COI check requested"));
+                            yield true;
```

Comment:
Look at you and your fancy new keywords ;)

Reply:
Second use in the repo, so not completely new. Old hat, in fact.